node/npm/glide/index.ts

```diff
@@ -185,19 +185,17 @@ function initialize() {
         SignedEncoding,
         UnsignedEncoding,
         UpdateByScore,
-        createLeakedArray,
-        createLeakedAttribute,
-        createLeakedBigint,
-        createLeakedDouble,
-        createLeakedMap,
-        createLeakedString,
-        parseInfoResponse,
```

Comment:
@avifenesh can you confirm that these functions can be removed? 
They were added as part of https://github.com/valkey-io/valkey-glide/commit/5eeb673fb729625da4c128bdc44beb68bf56cda0#diff-90759696d403370b4263a25799d215fe2cd1471e7ae440426c8177f636aa5bac, but it's unclear why they are needed (needed for alpine?)

Reply:
Note.  These functions are _only_ used in tests.  We should consider reworking this.  Maybe using a compile flag to generate these functions for test only.  They do not need to be added to production code.  
Also note: `createLeakedStringVec` _is_ used in production code, however this function is not exported from the package module.

Reply:
we cant remove it without fixing the release tests. For now, we can leave it as is . so I created a new PR to add the missing exports https://github.com/valkey-io/valkey-glide/pull/2342